### PR TITLE
doc: add minutes for meeting 2016-07-14

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: wg-meetings/2016-07-14.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+* Sampson Gao
+  * Modify the init method for addons by NODE_MODULE_ABI
+  * building new tests for api/abi
+* Ali Ijaz Sheikh (@ofrobots)
+  * Steady progress on FFI proof-of-concept.
+
+## Actions from last meeting
+Michael to review README.md to see if we can improve positioning versus Nan wg.  Rolled over as he has not gotten to this yet.
+
+## Upate on ABI stable working
```
Comment (ianwjhalliday) [inline wg-meetings/2016-07-14.md]: type `Update`

File: wg-meetings/2016-07-14.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+## Keep working in Ian's fork ?
+
+Quick discussion on whether we should keep working in Ian's fork or not. Agreed
+we should move discussion back to github.  Michael has action to open issues
+to start that disscussion.
+
+## Open issues review
+
+The other action items too more than the allocated time (30 mins over). So
```
Comment (ianwjhalliday) [inline wg-meetings/2016-07-14.md]: typo `took more`

File: wg-meetings/2016-07-14.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+https://github.com/ianwjhalliday/node/tree/api-opaque-prototype/src
+
+  * node_jsvmapi.cc
+  * node_jsvmapi.h
+  * node_jsvmapi_internal.h
+  * node_jsvmapi_types.h
+
+and NaN examples here:
+
+https://github.com/ianwjhalliday/node-addon-examples/tree/api-opaque-prototype
```
Comment (ianwjhalliday) [inline wg-meetings/2016-07-14.md:63]: we should probably also include a link to the leveldown fork

https://github.com/ianwjhalliday/leveldown/tree/api-opaque-prototype

File: wg-meetings/2016-07-14.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+## Upate on ABI stable working
+
+* walked through node_jsvmapi.h to show current methods
+* walked through selection of NaN examples with comparison between
+  ABI stable and NaN/native versions.
+* Walked through level down as a larger module that has been ported observer
+* Discussed some next steps which include:
+  * Updating EPS to reflect current methods
+  * Opening issue (Michael) to discuss if we should continue to use Ian's
+  * repo or move code somewhere else.
```
Comment (ianwjhalliday) [inline wg-meetings/2016-07-14.md]: the asterisk here should be removed so that this doesn't become a new bullet

**Comment by ianwjhalliday** [comment]
Other than some minor typos and formatting issues this looks great to me

**Comment by mhdawson** [comment]
Pushed commit to address first round of comments.  Will give it another day or so and if no additional comments go ahead and land base on Ian's lgtm.

**Comment by mhdawson** [comment]
Landed as d293ebcaba2dcca436fe3f76e8be43f662ca1ee9

**Pull request closed without merging**